packages/react/src/components/RadioTile/index.ts

```diff
@@ -5,7 +5,9 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import RadioTile from './RadioTile';
+import RadioTile, { RadioTileProps } from './RadioTile';
 
 export default RadioTile;
 export { RadioTile };
+
+export type { RadioTileProps };
```

Comment:
I noticed that many of the recently added types don't export their interface. I'm not sure if there is a good reason not to, but I decided to include it.

This matches the [original implementation from DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/carbon-components-react/lib/components/RadioTile/RadioTile.d.ts) and the current [export for `Checkbox`](https://github.com/carbon-design-system/carbon/blob/main/packages/react/src/components/Checkbox/index.ts)